certified-connectors/DocJuris/apiDefinition.swagger.json

```diff
@@ -670,6 +670,68 @@
         ]
       }
     },
+    "/v1/subscriptions/screeningReportUpdated": {
+      "x-ms-notification-content": {
+        "description": "ScreeningReportUpdatedMessage",
```

Comment:
@joshuaapple descriptions should be a valid English string, please make sure that this description looks like a sentence. More information about English strings can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).